homeassistant/components/heatmiser/manifest.json

```diff
@@ -2,9 +2,7 @@
   "domain": "heatmiser",
   "name": "Heatmiser",
   "documentation": "https://www.home-assistant.io/integrations/heatmiser",
-  "requirements": [
-    "heatmiserV3==0.9.1"
-  ],
+  "requirements": ["heatmiserV3==1.1.10", "pyserial-asyncio==0.4"],
```

Comment:
Why is `pyserial-asyncio` a requirement? It doesn't seem to be used and it is already [pulled-in](https://github.com/andylockran/heatmiserV3/blob/master/setup.py#L26) by `heatmiserV3`.

Reply:
Have removed - wasn't previously part of the upstream lib, but I've added it.